### DISCUSSION TITLE
How do you use includes when using in browser?

### DISCUSSION THREAD

**Discussion by nboliver** [harttle/liquidjs#485] [Q&A]
The [partials docs](https://github.com/harttle/liquidjs/wiki/Partials-and-Layouts) and the browser demo don't explain how to set up the file paths for includes when using in the browser. Does anyone have an example of how to do this?

**Comment by harttle** [comment]
The actual url is resolved based on [the root option](https://harttle.land/liquidjs/interfaces/_liquid_options_.liquidoptions.html#root), the only difference with Node.js is there can be at most one such `root` (it should be an string insteadof an array of string) and templates are fetched without lookup all roots.

In addition there's an abstract filesystem API available to define how to fetch the template, see https://github.com/harttle/liquidjs/wiki/Render-a-File#abstract-file-system

The actual PR for this feature is: https://github.com/harttle/liquidjs/pull/57 There're some integration tests, hopefully I can write a specific doc for this in the future.

**Comment by harttle** [comment]
A more descriptive wiki for partial lookups is updated, see https://github.com/harttle/liquidjs/wiki/Render-a-File#template-lookup
